src/PlusWidgets/CMakeLists.txt

```diff
@@ -45,8 +45,23 @@ SET(${PROJECT_NAME}_INCLUDE_DIRS
 IF(NOT (CMAKE_VERSION VERSION_LESS "3.8"))
   # CMake 3.8 and later puts generated headers (such as ui_QPlusDeviceSetSelectorWidget.h) into
   # .../PlusLib-bin/src/PlusWidgets/PlusWidgets_autogen/include
-  LIST(APPEND ${PROJECT_NAME}_INCLUDE_DIRS ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}_autogen/include)
+  
+  # CMake 3.10 puts generated headers (such as ui_QPlusDeviceSetSelectorWidget.h) into
+  # .../PlusLib-bin/src/PlusWidgets/PlusWidgets_autogen/include_<Config>
+
+  IF(IS_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}_autogen/include)

```

Comment:
This hardcodes only the 3 configurations.

Can you throw this into a loop that iterates over the entries in CMAKE_CONFIGURATION_TYPES?